Remove CFLAGS and LDFLAGS deduplication code which prevents universal builds from succeeding.
http://sourceforge.net/p/graphicsmagick/bugs/320/
http://hg.code.sf.net/p/graphicsmagick/code/rev/62387f365b84
--- configure.orig	Sun Oct 04 12:42:33 2015 -0500
+++ configure	Mon Oct 05 08:44:32 2015 -0500
@@ -30328,50 +30328,6 @@
 
 
 #
-# Deduplicate LDFLAGS
-#
-LDFLAGS_NEW=''
-for flag in $LDFLAGS
-do
-    add=yes
-    for test_flag in $LDFLAGS_NEW
-    do
-        if test $flag = $test_flag
-        then
-            add=no
-            break
-        fi
-    done
-    if test $add = yes
-    then
-        LDFLAGS_NEW="$LDFLAGS_NEW $flag"
-    fi
-done
-LDFLAGS=$LDFLAGS_NEW
-
-#
-# Deduplicate CFLAGS
-#
-CFLAGS_NEW=''
-for flag in $CFLAGS
-do
-    add=yes
-    for test_flag in $CFLAGS_NEW
-    do
-        if test $flag = $test_flag
-        then
-            add=no
-            break
-        fi
-    done
-    if test $add = yes
-    then
-        CFLAGS_NEW="$CFLAGS_NEW $flag"
-    fi
-done
-CFLAGS=$CFLAGS_NEW
-
-#
 # Remove extraneous spaces from output variables (asthetic)
 #
 X_CFLAGS=`echo $X_CFLAGS | sed -e 's/  */ /g'`
